phpBB/styles/prosilver/theme/stylesheet.css

```diff
@@ -7,6 +7,7 @@
     --------------------------------------------------------------
 */
 
+@import url("https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css");
```

Comment:
Should be added to our assets instead, so it also works if your forum is not connected to the internet or e.g. CDN is disabled?